scripts/vendors/shared.psm1

```diff
@@ -150,21 +150,32 @@ function Get-GHDLBinary
 		[string]$GHDL
 	)
 
-	if ($GHDL -ne "")
-	{	$GHDLBinary = $GHDL       }
-	elseif (Test-Path env:GHDL)
-	{	$GHDLBinary = $env:GHDL   }
-	else
-	{	try
-		{	write-host "calling which ..."
-			$GHDLBinary = (Get-Command "ghdl.exe").Source }
-		catch
-		{	Write-Host "Use adv. options '-GHDL' to set the GHDL executable." -ForegroundColor Red
-			Exit-CompileScript -1
+	if ($GHDL -eq "")
+	{	$GHDL = $env:GHDL	}
+
+
+	if ($GHDL -ne "") {
+		if (Test-Path $GHDL -PathType Container) # is directory
```

Comment:
I think it is an interesting feature to support GHDL either being either a directory or a binary. However, that would introduce an inconsistency with the bash scripts and pyGHDL.

Reply:
I have removed the feature and just fixed the docstring and the bug where `Get-Command` is called.